playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -98,6 +98,21 @@
   tasks:
   - set_fact:
       openshift_rpm: "{{ hostvars[groups['nodes'][0]]['yum'].results | selectattr('name', 'match', '^(origin|atomic-openshift)$') | first }}"
+  - name: fetch subscription id
+    command: az account show -o json --query id
+    register: subscription_id
```

Comment:
should be able to use the AZURE_SUBSCRIPTION_ID environment variable

Reply:
I'm fine either way.  Felt like this was safer as it queried instead.  Will update to env var.